test/suites/container_devices_nic_bridged.sh

```diff
@@ -38,6 +38,7 @@ test_container_devices_nic_bridged() {
   lxc profile device set "${ctName}" eth0 limits.egress 2Mbit
   lxc profile device set "${ctName}" eth0 host_name "${vethHostName}"
   lxc profile device set "${ctName}" eth0 mtu "1400"
+  lxc profile device set "${ctName}" eth0 txqueuelen "1200"
```

Comment:
Can you expand this check to check it actually gets applied by looking in /sys (I think)?

Reply:
Done